plugins/se.infomaker.publishflow/getExternalChange.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+import { api } from 'writer'
+
+function getItemMetaExtPropertyValue(type) {
+    switch (type) {
+        case 'imext:pubstart':
+            return {
+                pubStart: api.newsItem.getPubStart()
+            }
+        case 'imext:pubstop':
+            return {
+                pubStop: api.newsItem.getPubStop()
+            }
+        case 'imext:haspublishedversion':
+            return {
+                hasPublishedVersion: api.newsItem.getHasPublishedVersion()
+            }
+        default:
+            return null
+    }
+}
+
+export default function getExternalChange(event) {
```

Comment:
We should probably avoid using default exports, and instead use `export {getExternalChange}` to improve intellisense